src/model_interface.cc

```diff
@@ -130,6 +130,8 @@ void ModelInterface::InitializePieces() {
     }
   }
 
+  user_defined_symbols.insert("<unk>");
```

Comment:
This has the effect that `<unk>` tokens in the raw input fed to `spm_encode` will be encoded as UNK and not tokenized to e.g. `< unk >`.

Reply:
This config should be used as reference whenever training the BPE models outside of the data repo